### ISSUE TITLE
The getter '_key' was called on null.

### ISSUE THREAD

**Issue by inosuke80171** [Stacked-Org/stacked#41]
Okay, first of all, great package! And I love your architecture design principle.

I followed your tutorial in YouTube, and I am having trouble implementing NavigationService, and I keep getting this error:

The getter '_key' was called on null.
Receiver: null
Tried calling: _key

The relevant error-causing widget was:
MyApp file:///D:/Documents/AndroidStudioProjects/testing_app/lib/main.dart

It's pointing me to NavigationService.navigatorKey. Do you have any idea what I might have been missing?

**Comment by FilledStacks** [comment]
Hey @inosuke80171 thanks. What flutter version are you on? I haven't seen that error on my side. Do a flutter pub outdated to see your current stacked_services package as well.

**Comment by inosuke80171** [comment]
Flutter version: 1.17.1

*on my pubspec:
```
  stacked: ^1.5.0
  auto_route: ^0.5.0
  stacked_services: ^0.2.3
  get_it: ^4.0.2
  injectable: ^0.3.0
```

I checked it, and I have all the latest version.

**Comment by FilledStacks** [comment]
It might be the new flutter release problem. I'll do an update now and then check, I'm still on 1.17.0

**Comment by inosuke80171** [comment]
Okay, awesome!

**Comment by FilledStacks** [comment]
Seeing the error now. Will fix then make a release

**Comment by FilledStacks** [comment]
Fixed in in 0.2.4 the newest version of get has a bug in it that I filed [here](https://github.com/jonataslaw/get/issues/105) with the workaround that I'm using in my code.

**Comment by inosuke80171** [comment]
Thanks for the quick fix! :)
